packages/mongodb-chatbot-server/src/routes/content/searchContent.ts

```diff
@@ -77,26 +79,55 @@ export function makeSearchContentRoute({
         searchResultsStore,
       });
     } catch (error) {
-      // TODO: error handling
+      throw makeRequestError({
+        httpStatus: 500,
+        message: "Unable to query search database",
+      });
     }
   };
 }
 
-// TODO: map FindContentResult to SearchContentResponseChunk
 function mapFindContentResultToSearchContentResponseChunk(
   result: FindContentResult
 ): SearchContentResponseBody {
-  // TODO:
   return {
-    results: [],
+    results: result.content.map(({ url, metadata, text }) => ({
+      url,
+      title: metadata?.pageTitle || "",
```

Comment:
```suggestion
      title: metadata?.pageTitle ?? "",
```

prefer nullish coalescing